_docs/examples/provider/README.md

```diff
@@ -158,6 +158,14 @@ echo "password" > key-pass.txt
 (cat key-pass.txt; cat key-pass.txt) | ./bin/akashctl keys export provider 2> key.txt
 ```
 
+### Configure Kubernetes Akash Namespace
+
+Create the `akash-services` namespace for running Provider.
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/ovrclk/akash/master/_docs/kustomize/namespace.yaml
+```
```

Comment:
can we not add `namespace.yaml` as a resource in `kustomization.yaml`?

Reply:
The docs are a little wishy-washy on this:
```
Setting the Namespace for all Resources
Reference:

The Namespace for all namespaced Resources declared in the Resource Config may be set with namespace. This sets the namespace for both generated Resources (e.g. ConfigMaps and Secrets) and non-generated Resources.
```
The field *sets* namespace, but AFAICT kustomize doesn't provide clear mechanisms to create the namespace with labels specific to just the namespace like we need.

Reply:
as discussed, that sets the namespace name for all resources that have the namespace.  agreed it's not clear if you can add a namespace resource, but it's worth trying.